public/js/dominant-colors-lazy-loading-public.js

```diff
@@ -42,8 +42,17 @@
 		image.classList.remove( 'dcll-placeholder' );
 	};
 
+	var firstUserInteraction = function (event) {
+		check();
+		document.body.removeEventListener('mousemove', firstUserInteraction, false );
```

Comment:
You should remove all listeners here, or firstUserInteraction will be called again.